### PR TITLE
DX: test that default config is not passed in RuleSet

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: src/RuleSet.php [hunk 1]
```diff
@@ -322,7 +320,7 @@ final class RuleSet implements RuleSetInterface
         ],
         '@PHPUnit50Migration:risky' => [
             '@PHPUnit48Migration:risky' => true,
-            'php_unit_dedicate_assert' => ['target' => PhpUnitTargetVersion::VERSION_5_0],
```
Comment (keradus) [inline src/RuleSet.php]: please revert
this is 50migration ruleset, and default target will not be 5_0 on 3.0

Comment (kubawerlos) [reply src/RuleSet.php]: How to modify test? Exclude all `@PHPUnitXXMigration` sets?

Comment (keradus) [reply src/RuleSet.php]: not really. need to think about best approach here

Comment (kubawerlos) [reply src/RuleSet.php]: So let's [ensure](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4433) the rule set are targeting proper PHPUnit's version.

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**